### DISCUSSION TITLE
ERROR: Invaliad svo file detected. This svo file appears to be corrupted or improperly formatted...

### DISCUSSION THREAD

**Discussion by plant1987** [zed-industries/zed#13222] [Feature Requests]
![image](https://github.com/zed-industries/zed/assets/71976453/f21c1189-144b-48e1-821b-43558c06a7ae)

Hi there:
    Does anyone get such kind of error? I am using the ZED mini camera for recording. If the recording time is longer than 15mins. Then the .svo2 file cannot be opened and cannot be exported to .AVI either. 
   However, I tried to record 10mins, then it it fine. I am using the SDK 4.1.2 version and H264 LOSSY(GPU). Anyone solve this error? Thanks a lot!

**Comment by SomeoneToIgnore** [comment]
Hey, this is a repository of a software tool, a text editor for programmers, not anyhow related to the device mentioned.
